### ISSUE TITLE
MR2 telemetry: Search term groups in history

### ISSUE THREAD

**Issue by eliserichards** [mozilla-mobile/fenix#22147]
## What questions will you answer with this data?

Implement the following telemetry probes: (might be an incomplete list. check with product)
- A user opens a search term group in history
- A user opens a tab from the search term group
- A user closes all tabs in the search term group
- A user closes a single tab in the search term group

## Acceptance Criteria
- [ ] ENG files a [DS JIRA](https://jira.mozilla.com/projects/DO/issues/DO-228?filter=allopenissues) request outlining their methodology.
- [ ] DS sign off on instrumentation methodology addressing product questions.
- [ ] Event pings can be queried via re:dash
- [ ] Event pings can be queried via amplitude
- [ ] We are sending telemetry events for the actions listed in the requirements
- [ ] We have documented the telemetry
- [ ] We have asked a data steward to [review](https://github.com/mozilla/data-review/blob/master/request.md) the telemetry

**Comment by Mugurell** [comment]
Seems like we already have a probe knowing when a search group is opened - https://dictionary.telemetry.mozilla.org/apps/fenix/metrics/history_search_term_group_tapped
Asked Kimmy to confirm the other three.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest Nightly build 96.0a1 from 11/19 using a Google Pixel 3a (Android 11)

✔️  A user opens a search term group in history (Previously done and checked in #22172) - Event Ping ac6c92f5-7ef7-4381-939e-3de638ab512c
```
{
          "timestamp": 4645,
          "category": "history",
          "name": "search_term_group_tapped"
        }
```
✔️  A user opens a tab from the search term group - Event Ping 7178208a-5338-4f91-94de-fbee9310ac64
```
{
          "timestamp": 1244,
          "category": "history",
          "name": "search_term_group_open_tab"
        }
```
✔️  A user closes all tabs in the search term group - Event Ping 7178208a-5338-4f91-94de-fbee9310ac64
```
{
          "timestamp": 14669,
          "category": "history",
          "name": "search_term_group_remove_all"
        }
```
✔️  A user closes a single tab in the search term group - Event Ping 7178208a-5338-4f91-94de-fbee9310ac64
```
{
          "timestamp": 13194,
          "category": "history",
          "name": "search_term_group_remove_tab"
        } 
```

[Logcat](https://github.com/mozilla-mobile/fenix/files/7568725/srcgrp.txt)
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/AJ1911002)
